libtransmission/README.md

```diff
@@ -1,15 +1,95 @@
 # Notes on the C-to-C++ Conversion
 
-- libtransmission was written in C for fifteen years, so eliminating all
-  Cisms is nearly impossible. **Modernization patches are welcomed** but
-  it won't all happen overnight. `tr_strdup()` and `constexpr` wil exist
-  side-by-side in the codebase for the forseeable future.
+- libtransmission was written in C for fifteen years, so eliminating all Cisms is nearly impossible. **Modernization
+  patches are welcomed** but it won't all happen overnight. `tr_strdup()` and `constexpr` wil exist side-by-side in the
+  codebase for the forseeable future.
 
-- It's so tempting to refactor all the things! Please keep modernization
-  patches reasonably focused so that they will be easy to review.
+- It's so tempting to refactor all the things! Please keep modernization patches reasonably focused so that they will be
+  easy to review.
 
 - Prefer `std::` tools over bespoke ones. For example, use `std::vector`
   instead of tr_ptrArray. Redundant bespoke code should be removed.
 
-- Consider ripple effects before adding C++ into public headers. Will it
-  break C code that #includes that header?
+- Consider ripple effects before adding C++ into public headers. Will it break C code that #includes that header?
```

Comment:
```suggestion
- Consider ripple effects before adding C++ into public headers. Will it break C code that #includes that header? If you think it might, [consult with downstream projects](https://github.com/transmission/transmission/issues/1826) to see if this is a problem for them.
```